crd-convert/src/main/java/io/strimzi/kafka/crd/convert/cli/AbstractCommand.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.cli;
+
+import io.strimzi.api.annotations.ApiVersion;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import picocli.CommandLine;
+
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+
+public abstract class AbstractCommand implements Runnable {
+    protected Logger log = LogManager.getLogger(getClass().getName());
+
+    @CommandLine.Spec
+    CommandLine.Model.CommandSpec spec;
+
+    @CommandLine.Option(names = {"-d", "--debug"}, description = "Use debug?")
+    boolean debug;
+
+    @CommandLine.Option(names = {"-ll", "--log-level"}, description = "Set log level to enable logging")
+    Level level;
+
+    protected void println(Object value) {
+        if (level != null) {
+            log.log(level, String.valueOf(value));
+        } else {
+            System.out.println(value);
+        }
+    }
+
+    public static class Versions implements Iterable<String> {
+        @Override
+        public Iterator<String> iterator() {
+            return Stream.of(
+                ApiVersion.V1ALPHA1,
+                ApiVersion.V1BETA1,
+                ApiVersion.V1BETA2,
+                ApiVersion.V1
```

Comment:
Again, let's not add it when we don't have it.

Reply:
Or isn't this the whole point, to be able at the end convert to v1 ?

Reply:
You're forgetting that there are versions in two dimensions. The metadata.apiVersion of the CRD resource itself (we want to move v1beta1 to v1) and the apiVersion of _our_ CRs (e.g. Kafka is currently v1beta1 and we're adding v1beta2) which are declared in the CRD via `spec.versions`.

Reply:
Ah, OK ... removing ...